base/test_views.py

```diff
@@ -1,8 +1,11 @@
-from django.test import TestCase
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
 from splinter import Browser
 
 
-class TestBaseViews(TestCase):
+test_url = 'http://localhost:8081/'
+
+
+class TestBaseViews(StaticLiveServerTestCase):
```

Comment:
Classes and functions should be documented according to the documentation style guide.

When documenting each test, the main thing you should consider is that the documentation should contain the information necessary to understand the test in case you break it in the future. For an example on how to do this, check this pull request; https://github.com/erikiado/jp2_online/pull/10, specifically the file perfiles_usuario/tests.py.